python/ray/tests/conftest.py

```diff
@@ -244,3 +244,25 @@ def log_pubsub():
     p.psubscribe(log_channel)
     yield p
     p.close()
+
+
+@pytest.fixture()
+def working_dir():
```

Comment:
Instead of making this as a fixture in the shared conftest, why not make it a helper method inside test_runtime_env.py?

Reply:
I can do that. But why not put it here? Should this potentially useful for other test cases?

Reply:
It's not relevant to other suites, so adding it in this common location will confuse readers who don't know about the purpose of this function.